segmentstore/contracts/src/main/java/io/pravega/service/contracts/AttributeUpdateType.java

```diff
@@ -40,7 +40,14 @@
     /**
      * Accumulates the new value to the existing attribute value (i.e., adds two numbers).
      */
-    Accumulate((byte) 3);
+    Accumulate((byte) 3),
```

Comment:
Can you take a look through these Javadocs. I think a few should be updated. Particularly None